packages/extension/src/companion/CompanionPopupButton.tsx

```diff
@@ -10,29 +8,52 @@ import AnalyticsContext from '@dailydotdev/shared/src/contexts/AnalyticsContext'
 import { CompanionPermission } from './CompanionPermission';
 import { useExtensionPermission } from './useExtensionPermission';
 
+export const COMPANION_ALERTS_LOCAL_KEY = 'companion:alerts';
+
+interface CompanionAlerts {
+  displayCompanionPopup?: boolean;
+}
+
+export const getCompanionAlerts = (): CompanionAlerts => {
```

Comment:
I think it's better to have this function in a separate file. Otherwise, when you import it in the background worker it might bloat its bundle size eventually